datamodel/app/create_app.py

```diff
@@ -9,6 +9,7 @@
 except ImportError:
     import psycopg2 as psycopg
 
+from gep_views.vw_tww_infiltration_installation import vw_tww_infiltration_installation
```

Comment:
Please move it to view.vw_tww_infiltration_installation gep_views is currently at the wrong place.

This should be reorganised in the future as :
views / vw_*
views / vw_tww_*
views / export / vw_*
views / SWMM/ vw_swmm_*
views / GWMP / vw_gwmp (gep is german) (only if we have the necessity to create some distinction)

Reply:
resolved in https://github.com/teksi/wastewater/pull/416/commits/91d6336daa466defef9a4acdfbb789ec52468b95